build.gradle

```diff
@@ -57,12 +58,15 @@ repositories {
 }
 
 dependencies {
+
+  annotationProcessor libraries.auto_value
+
   compile libraries.guava,
     libraries.jsr305,
     libraries.javax_annotations
 
-  compileOnly libraries.autovalue,
-    libraries.error_prone_annotations
+  compileOnly libraries.error_prone_annotations,
+    libraries.auto_value_annotations
```

Comment:
I think auto_value is the compile only dependency and auto_value_annotations is an actual dependency

Reply:
done for annotation. auto_value should be on the annotationProcessor path.